build

```diff
@@ -0,0 +1 @@
+Subproject commit 28b8eedd19c62e0adce123536fea9642ef376336
```

Comment:
this is https://github.com/caarlos0/shell-ci-build as a submodule to provide us shellcheck